docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md

```diff
@@ -1,32 +1,40 @@
 ---
 id: 1password-source
+slug: /c2c/1password
 title: 1Password Source
-sidebar_label: 1Password
```

Comment:
@JV0812 , could you add this line again? Otherwise it's going to be looking weird compared to the rest of C2Cs.

Reply:
@yleiferman I have added this.